client_code/historic/logging.py

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2021 anvilistas
+from anvil_extras import logging
+
+__version__ = "0.0.1"
+
+LOGGER = logging.Logger("anvil_labs.historic", level=logging.INFO)
+
+
+def set_debug(with_debug):
+    levels = {True: logging.DEBUG, False: logging.INFO}
+    LOGGER.setLevel(levels[with_debug])
```

Comment:
We didn't add this api in the anvil-extras logger.

Reply:
oops. that's copilot not quite getting it right and me not paying enough attention!